src/features/defi/helpers/normalizeOpportunity.tsx

```diff
@@ -107,18 +107,21 @@ type NormalizeOpportunitiesProps = {
   cosmosSdkStakingOpportunities: MergedActiveStakingOpportunity[]
   foxEthLpOpportunity?: EarnOpportunityType
   stakingOpportunities?: EarnOpportunityType[]
+  saversVaults?: EarnOpportunityType[]
```

Comment:
This should live under the `stakingOpportunities` opportunity type

Reply:
I didn't want to muck too much with this, since this is just a way to get something to click on to bring up the modal.

Reply:
Yeah fine with that since there's really no data for it in the store yet